docs/lab/aws/dkms-silly-driver.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
+kind: Module
+metadata:
+  name: silly-dkms
+spec:
+  driverContainer:
+    container:
+      modprobe:
+        moduleName: silly
+      kernelMappings:
+        - literal: 5.4.196-108.356.amzn2.x86_64
+          containerImage: 804517629211.dkr.ecr.eu-west-3.amazonaws.com/silly-kmod:1.7
```

Comment:
Can you please remind me what is the difference between `silly-kmod` and `simple-kmod`?